src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Circle implements Figure {
+    private int radius;
+
+    public Circle(String color, int radius) {
+        this.radius = radius;
+    }
+
+    @Override
+    public double getArea() {
+        return Math.PI * radius * radius;
+    }
+}
```

Comment:
The class is missing a method to return the color or draw the figure, which is expected for a class implementing the Figure interface.